packages/medusa/src/api/routes/admin/promotions/[id]/route.ts

```diff
@@ -0,0 +1,16 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IPromotionModuleService } from "@medusajs/types"
+import { MedusaRequest, MedusaResponse } from "../../../../../types/routing"
+
+export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
```

Comment:
love it 😻

Reply:
would it work if we do `export const function GET` ?

Reply:
yes it does, but I'd like to keep it consistent with the ones from the docs - https://docs.medusajs.com/development/api-routes/create

Reply:
got it 👍